### ISSUE TITLE
svnDirectoryName is never set to anything.

### ISSUE THREAD

**Issue by JeremyMorton** [mayerwin/vs-customize-window-title#41]
I have SVN binaries directory set to "C:\Program Files\TortoiseSVN\bin" (and it's on my path), but my solution files under SVN managed directories have never set svnDirectoryName.

**Comment by mayerwin** [comment]
Hi Jeremy, is this a new issue (i.e. was it working well before and no longer)? Which version of Visual Studio and the extension are you using? 
It'd also help if you could include the pattern that is being applied.

**Comment by JeremyMorton** [comment]
No, it has never worked for me over the last ~6-9 months that I have used this. I am seeing this with VS 2017 and VS 2019, and I'm currently on version 4.3.0 of this plugin.

Here's a sample string:
`[[path:2]] [solutionName] | [svnDirectoryName][gitBranchName] - [ideName] [vsMajorVersionYear] - [configurationName]`

I have both Git projects and SVN projects, which is why I have `[svnDirectoryName][gitBranchName]`.

**Comment by mayerwin** [comment]
Thanks for the clarifications. When you say: "svnDirectoryName is never set to anything", do you mean the tag is replaced by an empty string?

Can you run C:\Program Files\TortoiseSVN\bin\svn.exe info when in your solution file directory (this is how the SVN directory name is retrieved)?

Other users are reporting that this tag works well, so I believe something must be incorrectly configured on your machine.
